lib/features/base/reloadable/reloadable_controller.dart

```diff
@@ -94,13 +97,23 @@ abstract class ReloadableController extends BaseController {
 
   void handleReloaded(Session session) {}
 
-  void _clearAllCacheAction() async {
-    await _cachingManager.clearAll();
-  }
-
-  void logoutAction() {
-    _deleteCredentialAction();
-    _clearAllCacheAction();
+  void logoutAction() async {
+    final authenticationType = _authorizationInterceptors.authenticationType;
+    if (authenticationType == AuthenticationType.oidc) {
+      await _logoutOidcInteractor.execute()
+        .then((value) async {
+          await Future.wait([
+              _deleteCredentialInteractor.execute(),
+              _cachingManager.clearAll(),
```

Comment:
I worried about these tasks will not be executed because `logoutOIDC` switch screen to web plugin in the flow.